Sources/Swift/Tools/SentryClientReport.swift

```diff
@@ -0,0 +1,25 @@
+@objc
+@_spi(Private) public final class SentryClientReport: NSObject, SentrySerializable {
+    
+    @objc(initWithDiscardedEvents:dateProvider:)
+    public init(discardedEvents: [SentryDiscardedEvent], dateProvider: SentryCurrentDateProvider) {
+        timestamp = dateProvider.date()
```

Comment:
`l`: So we consistently use self in the init method.
```suggestion
        self.timestamp = dateProvider.date()
```

Reply:
I'm used to the airbnb swift style guide here: https://github.com/airbnb/swift?tab=readme-ov-file#omit-self

No strong preference other than that if we have a preference we should make it automatically format that way (either auto add or auto remove, or no preference)